localstack/services/es/es_starter.py

```diff
@@ -52,8 +52,8 @@ def start_elasticsearch(port=None, version=None, delete_data=True, asynchronous=
     # hence we use the default bind address 127.0.0.0 and put a proxy in front of it
     cmd = (('%s/bin/elasticsearch ' +
         '-E http.port=%s -E http.publish_port=%s -E http.compression=false ' +
-        '-E path.data=%s') %
```

Comment:
Can we please use the `TMP_FOLDER` config variable here, instead of hardcoding the path here:
```
    backup_dir = os.path.join(config.TMP_FOLDER, 'es_backup')
    cmd = (('%s/bin/elasticsearch ' +
        '-E http.port=%s -E http.publish_port=%s -E http.compression=false ' +
        '-E path.data=%s -E path.repo=%s') %
        (base_dir, backend_port, backend_port, es_data_dir, backup_dir))
```

Reply:
I'm not seeing where `backup_dir` is defined. Do you mean `config.TMP_FOLDER`?

Reply:
Yes, that's right. Please see this line in the comment above:
```
    backup_dir = os.path.join(config.TMP_FOLDER, 'es_backup')
```
The variable `config.TMP_FOLDER` points to the path of a local temporary folder, and here we would define `es_backup` as the folder `"es_backup"` within the local temp folder.

Hope that makes sense, please let me know if anything is unclear. Thanks